test/Quaternion.jl

```diff
@@ -522,7 +522,13 @@ Base.:(/)(a::MyReal, b::Real) = a.val / b
             for _ in 1:100
                 q1 = nquatrand()
                 q2 = qrotation(rotationmatrix(q1), q1)
+                q3 = qrotation(rotationmatrix(q1))
                 @test q1 ≈ q2
+                @test q2 === q3 || q2 === -q3
```

Comment:
Why is this check included?

Reply:
The range of `qrotation(dcm)` is {w≥0}, but the range of `qrotation(dcm, q)` is not. This test confirms that `qrotation(dcm, q)` differs only in sign from `qrotation(dcm)`.